0.23.0 release (#1067)
packit/packit: CHANGELOG.md

# 0.23.0

> - The `propose-update` has been deprecated in favour of `propose-downstream`

Reviewer: ```suggestion
- The `propose-update` has been renamed to `propose-downstream`; `propose-update` is now deprecated
```

Reviewer: oh, it's 2 lines actually, so my suggestion doesn't make sense; my point here is that we renamed the command; not sure if my suggestion will be more clear to users

Reviewer 2: Yes, I think Tomas's version is a bit more clearer

  to unify the naming between CLI and service. ([@jpopelka](https://github.com/jpopelka), [#1065](https://github.com/packit-service/packit/pull/1065))
- Our README has been cleaned and simplified. ([@ChainYo](https://github.com/ChainYo), [#1058](https://github.com/packit-service/packit/pull/1058))
- The :champagne: comment with the installation instructions has been disabled by default. ([@mfocko](https://github.com/mfocko), [#1057](https://github.com/packit-service/packit/pull/1057))
  - More information can be found in [our documentation](https://packit.dev/docs/configuration/#notifications).

> - The packit is being prepared to be released in RHEL/EPEL. ([@nforro](https://github.com/nforro), [#1055](https://github.com/packit-service/packit/pull/1055))

Reviewer: ```suggestion
- Packit is being prepared to be released in EPEL 8 so it can be consumed in RHEL and CentOS Stream. ([@nforro](https://github.com/nforro), [#1055](https://github.com/packit-service/packit/pull/1055))
```
